aat/src/aat/java/uk/gov/hmcts/ccd/definitionstore/tests/functional/elasticsearch/ElasticsearchImportDefinitionTest.java

```diff
@@ -64,13 +69,29 @@ void shouldImportValidDefinitionMultipleTimes() {
             .when()
             .post("/import");
 
-        verifyIndexAndFieldMappings(true);
+        verifyIndexAndFieldMappingsWithRetries(true);
     }
 
-    private void verifyIndexAndFieldMappings(boolean verifyNewFields) {
-        ValidatableResponse response = getIndexInformation(CASE_INDEX_ALIAS);
-        verifyIndexAndAlias(response);
-        verifyCaseDataFields(response, verifyNewFields);
+    private void verifyIndexAndFieldMappingsWithRetries(boolean verifyNewFields) {
```

Comment:
you could call make this method more generic by calling it `withRetries` and passing a Java function as argument and a String description:
`withRetries(() -> verifyIndexAndFieldMappings(verifyNewFields), "ES index verification")

Reply:
added withRetries in a utility class